complete/_rg

```diff
@@ -45,6 +45,7 @@ _rg() {
     '--ignore-file=[specify additional ignore file]:file:_files'
     '(-v --invert-match)'{-v,--invert-match}'[invert matching]'
     '(-n -N --line-number --no-line-number)'{-n,--line-number}'[show line numbers]'
+    '(--line-number-width)'{-l+, --line-number-width=}'[specify width of displayed line number]:number of columns'
```

Comment:
The 'exclusive-or' list at the beginning should exclude all options that this one makes irrelevant. So we would probably want something like this at the very least:

```
(-l -N --line-number-width --no-line-number)
```

Then the definition for `-N` should be updated similarly to exclude `--line-number-width`

(I think we should also add a few more, like `-c`, but it seems i haven't accounted for them in the past)

Reply:
Oh, i was going to comment that i don't think a short option is necessary, but it seems you reached that conclusion too, since you didn't actually add one. You should remove the `-l+` then (which is used by something else anyway). So it'd be just one string, `(...)--line-number-width[...]:...`

Reply:
Ah, it is with some embarrassment that I have to admit I was very excited by raising what would be my first PR in an OSS project that I failed to get this part of the change absolutely right. 😞  I'm sorry about that. Thanks for clarifying. I will make the changes you suggested.

And yes, I also felt like we didn't need a short option for this. I briefly considered using `-w` but decided against it.